javascript/hand-gesture-recognizer/gestures.js

```diff
@@ -0,0 +1,340 @@
+const gestures = {
```

Comment:
What do you think we add a comment describing what is the purpose of this file and a reference to the documentation or instructions?

Reply:
Sure! I have updated it with a small description